src/EditorFeatures/Core/FindUsages/AbstractFindUsagesService.SymbolMoniker.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.SymbolMonikers;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Shared.Utilities;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.FindUsages
+{
+    using static FindUsagesHelpers;
+
+    internal abstract partial class AbstractFindUsagesService
+    {
+        private static async Task FindSymbolMonikerReferencesAsync(
+            IFindSymbolMonikerUsagesService monikerUsagesService,
+            ISymbol definition,
+            IFindUsagesContext context,
+            CancellationToken cancellationToken)
+        {
+            if (monikerUsagesService == null)
+                return;
```

Comment:
The file is null annotated and this is internal, why do this? @tmat was generally removing these if it's now ensured by the compiler, assuming we've annotated all the callers.

Reply:
yes, removed.